content/types/_client_options.textile

```diff
@@ -14,28 +14,29 @@ h4.
+- <span lang="default">clientId</span><span lang="python">client_id</span><span lang="ruby">:client_id</span> := A client ID, used for identifying this client when publishing messages or for presence purposes. The <span lang="default">@clientId@</span><span lang="ruby">@client_id@</span> can be any non-empty string. This option is primarily intended to be used in situations where the library is instanced with a key; note that a <span lang="default">@clientId@</span><span lang="ruby">@client_id@</span> may also be implicit in a token used to instance the library; an error will be raised if a <span lang="default">@clientId@</span><span lang="ruby">@client_id@</span> specified here conflicts with the <span lang="default">@clientId@</span><span lang="ruby">@client_id@</span> implicit in the token. "Find out more about client identities":/how-ably-works#client-identity<br>__Type: @String@__
 
-- <span lang="default">authCallback</span><span lang="ruby">:auth_callback</span> := A <span lang="default">function</span><span lang="java">@TokenCallback@ instance</span><span lang="ruby">proc / lambda</span> to call when a new token is required. The role of the callback is to either generate a signed "@TokenRequest@":/realtime/types#token-request which may then be submitted automatically by the library to the "Ably REST API @requestToken@":/rest-api#request-token; or to provide a valid token in as a "@TokenDetails@":/realtime/types#token-details object. See "authentication":/rest/authentication for details of the token request format and associated API calls.<br>__Type: <span lang="default">@Function@</span><span lang="java">@TokenCallback@</span><span lang="ruby">@Proc@</span>__
+- <span lang="default">authCallback</span><span lang="python">auth_callback</span><span lang="ruby">:auth_callback</span> := A <span lang="default">function</span><span lang="java">@TokenCallback@ instance</span><span lang="python">lambda</span><span lang="ruby">proc / lambda</span> to call when a new token is required. The role of the callback is to either generate a signed "@TokenRequest@":/realtime/types#token-request which may then be submitted automatically by the library to the "Ably REST API @requestToken@":/rest-api#request-token; or to provide a valid token in as a "@TokenDetails@":/realtime/types#token-details object. See "authentication":/rest/authentication for details of the token request format and associated API calls.<br>__Type: <span lang="default">@Function@</span><span lang="java">@TokenCallback@</span><span lang="ruby">@Proc@</span>__
```

Comment:
On double-checking, apparently in python you can pass either a named function or a lambda, and they can be called the same way. So I think I should probably change 'lambda' to just 'function' at the beginning of the para to just 'function', and assume that python devs will know what I mean